src/diagnostics/OpenPMDWriter.cpp

```diff
@@ -57,21 +57,24 @@ OpenPMDWriter::InitDiagnostics (const int output_step, const int output_period,
     if (output_period < 0 ||
        (!(output_step == max_step) && output_step % output_period != 0)) return;
 
+    m_outputSeries.resize(nlev+1);
+    m_last_output_dumped.resize(nlev+1);
```

Comment:
Why is it `.resize(nlev+1)` and not `.resize(nlev)` ?

Reply:
Thanks! That's a typo, it is of course either `max_lev + 1` or `nlev`